src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -136,7 +149,7 @@ explanation.
 Another case where the use of `#` is handy is when you want to ignore
 error handling. Lets say you want the following,
 
-```rust,ignore
+```rust
```

Comment:
Note that taking the `ignore` off of this keeps making the test fail, because it's a doc comment that doesn't document anything.

Reply:
Erf, good point!